pkg/logger/logger_redis.go

```diff
@@ -0,0 +1,65 @@
+package logger
+
+import (
+	"github.com/gomodule/redigo/redis"
+	"github.com/rs/zerolog/log"
+)
+
+// ContainerLoggerRedis send stdout/stderr to a
+// Redis PubSub channel
+type ContainerLoggerRedis struct {
+	ContainerLogger
+
+	Endpoint      string
+	ChannelStdout string
+	ChannelStderr string
+
+	conn redis.Conn
+}
+
+// NewContainerLoggerRedis create new redis backend and initialize connection
+func NewContainerLoggerRedis(endpoint string, channel string) (*ContainerLoggerRedis, error) {
+	log.Debug().Str("endpoint", endpoint).Str("channel", channel).Msg("initializing redis logging")
+
+	c, err := redis.DialURL(endpoint)
+	if err != nil {
+		return &ContainerLoggerRedis{}, err
+	}
+
+	return &ContainerLoggerRedis{
+		Endpoint:      endpoint,
+		ChannelStdout: channel,
+		ChannelStderr: channel,
+		conn:          c,
+	}, nil
+}
+
+// Stdout handle a stdout single line
+func (c *ContainerLoggerRedis) Stdout(line string) error {
+	_, err := c.conn.Do("PUBLISH", c.ChannelStdout, line)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Stderr handle a stderr single line
+func (c *ContainerLoggerRedis) Stderr(line string) error {
+	_, err := c.conn.Do("PUBLISH", c.ChannelStderr, line)
```

Comment:
Shouldn't we have a different channel for stderr or at least prefix the line with "E:" the same way u do with console logger?

Reply:
We support a different channel, it's just not exposed publicly on the reservation.

Reply:
Now you can specify different channel for stdout and stderr